server/src/main/scala/actors/persistent/nebo/NeboArrivalActor.scala

```diff
@@ -0,0 +1,89 @@
+package actors.persistent.nebo
+
+import actors.persistent.nebo.NeboArrivalActor.getRedListPassengerFlightKey
+import actors.persistent.staffing.GetState
+import actors.persistent.{RecoveryActorLike, Sizes}
+import actors.serializers.NeboArrivalMessageConversion
+import akka.actor.Props
+import akka.persistence.{Recovery, SaveSnapshotSuccess, SnapshotSelectionCriteria}
+import drt.shared.CrunchApi.MillisSinceEpoch
+import drt.shared.{NeboArrivals, RedListPassengers, SDateLike}
+import org.slf4j.{Logger, LoggerFactory}
+import scalapb.GeneratedMessage
+import server.protobuf.messages.NeboPassengersMessage.NeboArrivalMessages
+
+object NeboArrivalActor {
+  def props(redListPassengers: RedListPassengers, now: () => SDateLike): Props =
+    Props(new NeboArrivalActor(redListPassengers, now, Option(now().millisSinceEpoch)))
+
+  def getRedListPassengerFlightKey(redListPassengers: RedListPassengers): String = {
+    s"${redListPassengers.flightCode.toLowerCase}-${redListPassengers.scheduled.getFullYear()}-${redListPassengers.scheduled.getMonth()}-${redListPassengers.scheduled.getDate()}-${redListPassengers.scheduled.getHours()}-${redListPassengers.scheduled.getMinutes()}"
```

Comment:
If we extract all the parts of this string to well named vals then it'll be easier to see what the string is made up from, eg like `s"$flightCode-$year-$month-$day-$hours-$minutes"`